tests/naive.py

```diff
@@ -1759,6 +1837,75 @@ def _total_diagonal_ndists(tile_lower_diag, tile_upper_diag, tile_height, tile_w
     return total_ndists
 
 
+def merge_topk_PI(PA, PB, IA, IB):
+    if PA.ndim == 1:
+        for i in range(PA.shape[0]):
+            if PB[i] < PA[i] and IB[i] != IA[i]:
+                PA[i] = PB[i]
+                IA[i] = IB[i]
+        return
+
+    k = PA.shape[1]
```

Comment:
Better  to use `if/return - else/return` instead of just `if / return`

Reply:
Yes, please